### ISSUE TITLE
Device can't wake after sleep.

### ISSUE THREAD

**Issue by xtrs84zk** [xtrs84zk/Aspire-V5-573P-Hackintosh#6]
After entering sleep mode, the wake process hangs and kernel panics itself after 180s. Logs here. 
[another_sleep_kp.txt](https://github.com/xtrs84zk/Aspire-V5-573P-Hackintosh/files/5322690/another_sleep_kp.txt)
[sleep_kp.txt](https://github.com/xtrs84zk/Aspire-V5-573P-Hackintosh/files/5322688/sleep_kp.txt)

**Comment by xtrs84zk** [comment]
Forgot to turn keepsyms=1 on the previous log.
[sleepkp3.txt](https://github.com/xtrs84zk/Aspire-V5-573P-Hackintosh/files/5343839/sleepkp3.txt)

**Comment by xtrs84zk** [comment]
Managed to wake up after sleep by reapplying ACPI hot patches on [c02b69e](https://github.com/xtrs84zk/Aspire-V5-573P-Hackintosh/commit/c02b69e03293e01bf22d8f7c8880f3131e89e683). However, due to diff between OC and Clover injection, I can't simply reuse those patches. Touchscreen and USB3 aren't detected with them. 
Didn't have a USB3 device on hand when I made the patch for OC, I'll get one and try again.
